### DISCUSSION TITLE
Delete Suspended Account

### DISCUSSION THREAD

**Discussion by Mario-Vella27** [community/community#142263] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi Everyone, 

I recently got my GitHub account suspended and I have no idea why. For this reason, I have a backup of all my data so I would like to delete this account so that I can use the email that the account had. Does anyone know how I can delete this account while it is suspended? Or should I just wait?

Thank you and I appreciate any help you can give me.

**Comment by stabldev** [comment]
hey @Mario-Vella27, I think you should wait for few.
Because, same thing happened to my (this) account, I send them an appeal via [this](https://support.github.com/contact/account). And I got their response and got my account back in like 1-2 days.
If it is your main account, I suggest you to wait. If not, create new one and move on.

**Reply by Mario-Vella27** [reply]
My problem isn't waiting, it's the fact that I want to use the email for the new account, as I will soon finish university and will lose access to my university email. So I need my personal email available, and not in the mood of creating a new one just for the sake of Github.

**Reply by stabldev** [reply, marked as answer]
I get it, but you can't do anything while it is suspended. Raise a ticket, I think they are processing tickets faster atm. change/delete once you get your account back.

> as I will soon finish university

soon means, in a week at most?

**Reply by tsukiyoz** [reply]
i use friend's github account for seeking help!
my account codegoz was suspended too, and i can do nothing to contact with github manager
i tried to submit a work order to the GitHub community, but there is no SMS in China, so I have nowhere to appeal!
i tried to submit help message to support@github.com which is a suggestion i found in internet. but due to regional restrictions, I am unable to send messages.


above all, i have no way to solve this problem. can any body help me?
i just want to do coding，use github.
i haven't do any bad things.

you can keep contacting with me by tsukiyo6@163.com
my account is [codegoz](github.com@codegoz)

**Reply by stabldev** [reply]
hey @lazywoo , you don't have to use another github account for submitting an appeal. your account is not removed, meaning you can raise ticket and you will get response to your email associated with the account.
i used [this](https://support.github.com/contact/cannot_sign_in), verify with your github account email then you will get redirected to another page with few more forms, fill them (select Can't Login from dropdowns). then you will get a mail saying ticket raised successfully.

thankfully, they fixed the issue and responded me within 2 days.

**Reply by MasterInQuestion** [reply]
https://github.com/orgs/community/discussions/141499
    https://github.com/orgs/community/discussions/142305
    .
    Your mileage depends.

**Comment by ercansahin16** [comment]
Response #2
